### ISSUE TITLE
Proportion vs. Percent

### ISSUE THREAD

**Issue by jacobbien** [radiant-rstats/radiant.multivariate#2]
Hi @vnijs !

In `summary.full_factor()` under "Fit Measures", it looks to me like the rows labeled "Variance %" and "Cumulative %" should be "Variance Prop" and "Cumulative Prop."

https://github.com/radiant-rstats/radiant.multivariate/blob/5ed7e4e48d23d11119582103ec32ec42be2fc9b5/R/full_factor.R#L220

Thanks!

**Comment by vnijs** [comment]
Not sure I follow @jacobbien. Are you suggesting the current labels are incorrect?

**Comment by jacobbien** [comment]
Yeah.  The computed quantities in those two rows of the table appear to be proportions rather than percentages.  So for example, when all of the variance is explained, it would say that the variance % is 1 rather than 100.

**Comment by vnijs** [comment]
Ah. I see what you mean now. Thanks for the suggestion @jacobbien. Will get this fixed. Same thing in pre-factor and perceptual maps.
